compliance_checker/base.py

```diff
@@ -325,12 +339,14 @@ def fix_return_value(v, method_name, method=None, checker=None):
     Transforms scalar return values into Result.
     """
     # remove common check prefix
-    method_name = (method_name or method.__func__.__name__).replace("check_",
-                                                                    "")
+    method_name = (method_name or method.__func__.__name__).replace("check_","")
     if v is None or not isinstance(v, Result):
         v = Result(value=v, name=method_name)
 
     v.name         = v.name or method_name
+    if v.name in ["high", "recommended", "suggested"]:
```

Comment:
Need to find a way to move this out of `base.py`. Can we somehow leverage the capability of calling a check using the 
`("name", bound_method)` model?